### DISCUSSION TITLE
How to set default commenstring for some files?

### DISCUSSION THREAD

**Discussion by c02y** [LunarVim/LunarVim#2799] [Q&A] [answered]
`:set commentstring?` for `/etc/pacman.conf` or `.gitconfig` or `.gitignore` is `; %s`

How to set such file to use `# %s` as the commentstring? I cannot make it work using something like
```
au BufNewFile,BufRead *.gitconfig set commentstring='# %s'
```

**Comment by uelei** [comment, marked as answer]
Hey @c02y using lvim rolling seems to be using `# %s`, but  any way, you can set it as it follows

```lua
local ft = require('Comment.ft')
ft.set('gitignore', '# %s')
ft.set('gitconfig', '# %s')
```

**Reply by c02y** [reply]
Thanks, I solved this problem long time ago

Here is what I'm using in `config.lua`:
```
require("Comment.ft")({ "", "gitconfig", "dosini", "org" }, "#%s")
```
